pages/api/v1/teams/[id]/roster/stats.ts

```diff
@@ -27,9 +30,13 @@ export default async (
   }
 
   let type: string;
-  if (shorttype === 'po' || shorttype === 'ps' || shorttype === 'rs') {
-    type = shorttype;
-  } else {
+  if (SeasonType === 'Pre-Season') {
+    type = 'ps';
+  }
+  else if (SeasonType === 'Playoffs') {
+    type = 'po';
+  }
+  else {
```

Comment:
Our query parameters should be case sensitive and this follows the review above.
```suggestion
  if (longType.toLowerCase().replace('-', '') === 'preseason') {
    type = 'ps';
  } else if (longType.toLowerCase() === 'playoffs') {
    type = 'po';
  } else {
```